packages/twenty-server/src/modules/connected-account/auto-companies-and-contacts-creation/utils/get-unique-contacts-and-handles.util.ts

```diff
@@ -0,0 +1,18 @@
+import { uniq, uniqBy } from 'lodash';
```

Comment:
We should avoid importing the whole lodash package from now on and use lodash.uniq, lodash.uniqby